Moggles/appsettings.json

```diff
@@ -1,18 +1,24 @@
 {
-    "Logging": {
-        "IncludeScopes": false,
-        "LogLevel": {
-            "Default": "Warning"
-        }
-    },
-    "ConnectionStrings": {
-
-    },
-    "CustomRoles": {
-    },
-    "Messaging": {
-    },
-    "ApplicationInsights": {
-        "InstrumentationKey": ""
+  "Logging": {
+    "IncludeScopes": false,
+    "LogLevel": {
+      "Default": "Warning"
     }
+  },
+  "ConnectionStrings": {
```

Comment:
I would recommend we remove ConnectionStrings all together from the base template, since new installs won't need it

Reply:
we can't until we remove the convert old data functionality

Reply:
I tested the application, and it will work with the connection string removed.  However, the convert page WILL fail (as expected).  Luckily the DI doesn't seem to kick in until you request it.